src/index.js

```diff
@@ -316,22 +393,19 @@ Dropzone.propTypes = {
   onDragEnter: React.PropTypes.func,
   onDragOver: React.PropTypes.func,
   onDragLeave: React.PropTypes.func,
-
   children: React.PropTypes.node, // Contents of the dropzone
   style: React.PropTypes.object, // CSS styles to apply
   activeStyle: React.PropTypes.object, // CSS styles to apply when drop will be accepted
   rejectStyle: React.PropTypes.object, // CSS styles to apply when drop will be rejected
   className: React.PropTypes.string, // Optional className
   activeClassName: React.PropTypes.string, // className for accepted state
   rejectClassName: React.PropTypes.string, // className for rejected state
-
   disablePreview: React.PropTypes.bool, // Enable/disable preview generation
   disableClick: React.PropTypes.bool, // Disallow clicking on the dropzone container to open file dialog
   onFileDialogCancel: React.PropTypes.func, // Provide a callback on clicking the cancel button of the file dialog
-
   inputProps: React.PropTypes.object, // Pass additional attributes to the <input type="file"/> tag
   multiple: React.PropTypes.bool, // Allow dropping multiple files
-  accept: React.PropTypes.string, // Allow specific types of files. See https://github.com/okonet/attr-accept for more information
+  accept: React.PropTypes.string, // Allow specific types of files. See https://github.com/okonet/attrccept for more information
```

Comment:
@okonet `https://github.com/okonet/attrccept` is invalid link and `https://github.com/okonet/attr-accept` is valid. Is it correct change?

Reply:
No, this is incorrect. Thanks for mentioning.